phpBB/includes/constants.php

```diff
@@ -225,10 +225,10 @@
 define('REFERER_VALIDATE_PATH', 2);
 
 // phpbb_chmod() permissions
-@define('CHMOD_ALL', 7);
-@define('CHMOD_READ', 4);
-@define('CHMOD_WRITE', 2);
-@define('CHMOD_EXECUTE', 1);
+@define('CHMOD_ALL', 7); // @deprecated 3.2
```

Comment:
Specify the actual version, e.g. `3.2.10` in this case.

Reply:
I followed the style in the other deprecated constants